feat(ins): add resize doc (#4972)
scaleway/docs-content: pages/instances/faq.mdx

Both are automatically updated when the Instance moves to another hypervisor, also changing its IPv6 and private IPv4. You can either use these hostnames directly or configure a [CNAME record](/dedibox-dns/concepts/#resource-records) linking to it.

### Can I resize my Instance?

> Yes, you can resize (also called vertical scaling) your Instance to adjust CPU, RAM, and storage capacity without the need of recreating it.

Reviewer: ```suggestion
Yes, you can resize (also called vertical scaling) your Instance to adjust CPU, RAM, and storage capacity without having to recreate it.
```

* To modify the CPU/RAM configuration of your Instance, refer to the [Instance resizing guide](/instances/how-to/resize-instance).

> * To increase the Block Storage volume of your Instance, refer to the [increasing Block Storage volume size gude](/block-storage/how-to/increase-block-volume/)

Author: ```suggestion
* To increase the Block Storage volume of your Instance, refer to the [increasing Block Storage volume size guide](/block-storage/how-to/increase-block-volume/)
```
